test/app.test.ts

```diff
@@ -1,23 +1,25 @@
+import { StatusCodes } from 'http-status-codes';
```

Comment:
Maybe we can add a regression test that specifically checks for the fixed bug? I.e. a test that passes different values for original and copy to the handler, and checks that the thrown exception has the correct item (id, path, ...)